map/gps_track.cpp

```diff
@@ -260,7 +232,7 @@ void GpsTrack::ProcessPoints()
 
   pair<size_t, size_t> addedIds;
   pair<size_t, size_t> evictedIds;
```

Comment:
Are these vars still needed? If yes, why?

Reply:
Added is used for the new Ids added from the location manager.

Evicted is used during the track clearing.
There were several sources of evicted Ids:
1. evictedIdsByDuration
2. evictedIdsByAdd
3. evictedIdsByClear

1 and 2 are removed because of refactoring and only the evictedIdsByClear is left.